docs/modules/ROOT/pages/apiserver/how-to.adoc

```diff
@@ -0,0 +1,25 @@
+= AppCat API Server
+
+Our AppCat API Server lets you query the state of AppCat services in the cluster. The operations can be performed through the kubectl command-line interface or you can also access the API directly using REST calls. Below is a list of available operations.
```

Comment:
Either provider an example of a REST call or remove the mention of it.

Reply:
Do we need a REST call to be documented?

Reply:
If we need it at any point we can add the docs for it then. But currently I don't think so. Most interaction with the API server will be through a proper k8s client anyway.

Reply:
I removed the REST reference